backend/src/main/java/com/huellapositiva/application/exception/ProposalNotPublic.java

```diff
@@ -0,0 +1,16 @@
+package com.huellapositiva.application.exception;
+
+/**
+ * This exception is thrown when a proposal is neither PUBLISHED nor FINISHED
+ * @see com.huellapositiva.domain.model.valueobjects.ProposalStatus
+ */
+public class ProposalNotPublic extends RuntimeException {
```

Comment:
ProposalNotPublicException?

Reply:
Ok, many thanks. This class has been deleted. There is other class with same exception, with "exception".